.github/workflows/ci.yaml

```diff
@@ -0,0 +1,22 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+      - name: Install dependencies
+        run: npm ci
+      - name: Run unit tests
+        run: npm run testing
```

Comment:
@cedpoilly, it doesn't _seem_ like the `test` command is relevant as I don't see a Karma configuration anywhere:

https://github.com/cedpoilly/vue-gapi/blob/1f6b1604f5ea0bde72b3775448f1d483d8cc9648/package.json#L25

Should we remove references to Karma and its dependencies for now?